### ISSUE TITLE
Mesos Integration Problems

### ISSUE THREAD

**Issue by mentalblock** [DataDog/puppet-datadog-agent#163]
Running `dd-agent info` gives me the following error.

...
Collector (v 5.6.3)
... 

    mesos
    -----
      - instance #0 [WARNING]
          Warning: This check is deprecated in favor of Mesos master and slave specific checks. It will be removed in a future version of the Datadog Agent.
          Warning: ('Connection aborted.', error(101, 'Network is unreachable'))
          Warning: ('Connection aborted.', error(101, 'Network is unreachable'))
          Warning: ('Connection aborted.', error(101, 'Network is unreachable'))
      - Collected 0 metrics, 0 events & 4 service checks

I noticed that the integration instructions in the webapp are inconsistent with the puppet module. To fix this, I had to rename the mesos.yaml file to mesos_slave.yaml. Please resolve this either in the agent or the Puppet module. Changing the agent is preferred because it probably isn't good practice to use the name of a configuration file for application logic. After changing the name of the YAML file and restarting the collector, I get:

    mesos_slave
    -----------
      - instance #0 [OK]
      - Collected 30 metrics, 0 events & 2 service checks

To summarize, the mesos integration does not work out of the box using this Puppet module and the problem lies within the agent but can be worked around by adding a slave/master parameter to the Puppet module to dictate what the configuration file should be named.

puppet-datadog-agent/manifests/integrations/mesos.pp

**Comment by truthbk** [comment]
@mentalblock thanks to @jangie and @flyinprogrammer contributions, this will be addressed in the upcoming puppet release! Thanks for your patience.

**Comment by truthbk** [comment]
This should've been addressed with #176 and the latest puppet module release (1.7.0).
